src/shared/state/pair.ts

```diff
@@ -0,0 +1,21 @@
+import { makeAutoObservable } from 'mobx';
+import { AssetSelectorValue } from '@penumbra-zone/ui/AssetSelector';
+
+class PairState {
+  from?: AssetSelectorValue;
+  to?: AssetSelectorValue;
```

Comment:
the idea of this store is to be used by other components like charts and order book